packages/now-cli/src/util/dev/server.ts

```diff
@@ -141,11 +141,14 @@ export default class DevServer {
   private updateBuildersPromise: Promise<void> | null;
   private updateBuildersTimeout: NodeJS.Timeout | undefined;
 
+  private environmentVars: Env | undefined;
+
   constructor(cwd: string, options: DevServerOptions) {
     this.cwd = cwd;
     this.debug = options.debug;
     this.output = options.output;
     this.envConfigs = { buildEnv: {}, runEnv: {}, allEnv: {} };
+    this.environmentVars = options.environmentVars || {};
```

Comment:
Something seems wrong here. We need the `envConfigs` to `options.environmentVars` if the user didn't provide any files.

Reply:
Happens in getNowConfig